jdaviz/configs/cubeviz/plugins/tests/test_parsers.py

```diff
@@ -152,9 +152,10 @@ def test_spectrum3d_no_wcs_parse(cubeviz_helper):
     assert data.shape == (2, 3, 4)  # x, y, z
     assert isinstance(data.coords, PaddedSpectrumWCS)
     assert_array_equal(flux.data, 1)
-    assert flux.units == 'nJy'
+    assert flux.units == 'nJy / pix2'
 
 
+@pytest.mark.skip(reason="unskip after 3192 merged")
```

Comment:
Do we know if #3192 going to be merged before or after this one?

Reply:
#3192 comes after and will unskip these